README.md

```diff
@@ -9,6 +9,9 @@ The following sample projects specify the main technologies used. A team that us
 
 * **[Sample 1: typescript-selenium-webdriver-sample](./typescript-selenium-webdriver-sample)**: 
   * Useful for teams using Selenium and TypeScript/JavaScript. 
+  
+* **[Sample 2: CSharpSeleniumWebdriverSample](./c%23-selenium-webdriver-sample)**: 

```

Comment:
I'd recommend avoiding the # in the folder name; it makes it easy to form links that involve accidental anchor tags. Probably `csharp-selenium-webdriver-sample`